src/packages.builds

```diff
@@ -1,11 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="BuildAndTest" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <_OriginalOSGroup Condition="'$(_OriginalOSGroup)' == ''">$(OSGroup)</_OriginalOSGroup>
+  </PropertyGroup>
+
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  
+  <PropertyGroup>
+    <FilterToOSGroup Condition="'$(FilterToOSGroup)' == '' and '$(_OriginalOSGroup)' == ''">$(OSEnvironment)</FilterToOSGroup>
```

Comment:
I know this was there before but is actually unclear to me if is is actually needed for packages. I know it is needed for tests.build but I'm less convinced it is needed for packages. 

@ericstj @chcosta do you guys know if we still actually need for the package builds?